images/scripts/bootc.setup

```diff
@@ -3,6 +3,11 @@ set -eux
 
 IMAGE="$1"
 
+# extend root file system to resized qcow2 image
+growpart /dev/vda 4
+mount -o remount,rw /sysroot
```

Comment:
Note: It's unfortunately not possible to immediately remount this `ro` after the resize2fs. That fails with "file system is busy" :cry:

Reply:
I feel like a suitable application of `fuser -m -k` might help here...

Reply:
I tried that already on my initial attempt, but this is too unspecific:
```
# fuser -v -m /sysroot
                     USER        PID ACCESS COMMAND
/sysroot:            root     kernel mount /sysroot
                     root          1 ....m systemd
                     root        734 ....m systemd-journal
                     root        743 ....m agetty
                     root        745 ....m systemd-udevd
                     root        747 ....m systemd-logind
                     root        748 ....m agetty
                     dbus        749 ....m dbus-broker-lau
                     dbus        750 ....m dbus-broker
                     root        767 ....m sshd
                     root        768 ....m sshd
                     root        772 ....m systemd
                     root        774 ....m (sd-pam
                     root        786 ....m sshd
                     root        787 ..c.m bash
                     root        791 ....m systemd-hostnam
# mount -o remount,ro /sysroot
mount: /sysroot: mount point is busy.
```
with `-k` it just kills the script itself (via killing ssh).

Reply:
→ Move this to bootc.bootstrap

Reply:
Done.